OSDOCS-4632: Observing network traffic (#54085)
openshift/openshift-docs: modules/network-observability-quickfilter.adoc

// Module included in the following assemblies:
//
// network_observability/observing-network-traffic.adoc
:_content-type: REFERENCE
[id="network-observability-quickfilter{context}"]
= Filtering the network traffic

> By default, the Network Traffic page displays the traffic flow data in the cluster based on the default filters configured in the FlowCollector instance. You can use the filter options to observe the required data by changing the preset filter. Following are the available filter options:

Reviewer: ```suggestion
By default, the *Network Traffic* page displays the traffic flow data in the cluster based on the default filters configured in the `FlowCollector` instance. You can use the filter options to observe the required data by changing the preset filter.
```

.Quick filters

> The default values in *Quick filters* drop-down are defined in the `FlowCollector` config. You can modify the options from console. 

Reviewer: ```suggestion
The default values in *Quick filters* drop-down menu are defined in the `FlowCollector` configuration. You can modify the options from console. 
```

> .Quick filters
> The default values in *Quick filters* drop-down are defined in the `FlowCollector` config. You can modify the options from console. 
> 
> .Advanced filters
> You can set the advanced filters by providing the parameter to be filtered and its corresponding text value. The section *Common* in the parameter drop-down list filters the results that match either *Source* or *Destination*. To enable or disable the applied filter, you can click on the applied filter listed below the filter options.

Reviewer: Suggestion: you might like using a [description list](https://docs.asciidoctor.org/asciidoc/latest/lists/description/#basic-description-list) here
```suggestion
Quick filters:: The default values in *Quick filters* drop-down are defined in the `FlowCollector` config. You can modify the options from console. 

Advanced filters:: You can set the advanced filters by providing the parameter to be filtered and its corresponding text value. The section *Common* in the parameter drop-down list filters the results that match either *Source* or *Destination*. To enable or disable the applied filter, you can click on the applied filter listed below the filter options.
```

Author: aah yes! That fits in better. Thanks for the suggestion!

[NOTE]
====
To understand the rules of specifying the text value, click *Learn More*.
====

.Query Options
You can use *Query Options* to optimize the search results, as listed below:

> ** *Reporter Node*: Every flow can be reported from both source and destination nodes. For cluster ingress, the flow is reported from the destination node and for cluster egress, the flow is reported from the source node. You can select either *Source* or *Destination*. The option *Both* is disabled for overview and topology view. The default selected value is *Destination*.

Reviewer: ```suggestion
** *Reporter Node*: Every flow can be reported from both source and destination nodes. For cluster ingress, the flow is reported from the destination node and for cluster egress, the flow is reported from the source node. You can select either *Source* or *Destination*. The option *Both* is disabled for *Overview* and *Topology* views. The default selected value is *Destination*.
```

> ** *Match filters*: You can determine the relation between different filter parameters selected in advanced filter. The available options are *Match all* and *Match any*. *Match all*  provides results that match all the values, and *Match any* provides results that match any of the values entered. The default value is *Match all*.

Reviewer: ```suggestion
** *Match filters*: You can determine the relation between different filter parameters selected in the advanced filter. The available options are *Match all* and *Match any*. *Match all*  provides results that match all the values, and *Match any* provides results that match any of the values entered. The default value is *Match all*.
```

> ** *Limit*: The limit for internal backend queries. Depending upon the matching and filter settings, several queries can be performed under the cover, each with this limit set, resulting in more results after aggregation.

Reviewer: Not sure what "under the cover" means here

Author: It was supposed to mean backend. But I have removed it now as the phrase doesn't make any relevance to the readers.

> You can click *Reset default filter* to remove the existing filters, and apply the filter defined in `FlowCollector` config.

Reviewer: ```suggestion
You can click *Reset default filter* to remove the existing filters, and apply the filter defined in `FlowCollector` configuration.
```
